### PR TITLE
8279547: [vectorapi] Enable vector cast tests after JDK-8278948

### PR DESCRIPTION
**Pull request by merykitty**
Hi,

This patch enables 2 vector cast operations that fail before.

Thank you very much.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8279547](https://bugs.openjdk.java.net/browse/JDK-8279547): [vectorapi] Enable vector cast tests after JDK-8278948

### Reviewers
 * [Vladimir Kozlov](https://openjdk.java.net/census#kvn) (@vnkozlov - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/6976/head:pull/6976` \
`$ git checkout pull/6976`

Update a local copy of the PR: \
`$ git checkout pull/6976` \
`$ git pull https://git.openjdk.java.net/jdk pull/6976/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 6976`

View PR using the GUI difftool: \
`$ git pr show -t 6976`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/6976.diff">https://git.openjdk.java.net/jdk/pull/6976.diff</a>

</details>

### CODE REVIEW

**Review by vnkozlov — COMMENTED** [review]
Good. Let me test it.

**Review by vnkozlov — APPROVED** [review]
Testing looks good.
It is trivial change. No need for second review.

**Comment by merykitty** [comment]
Thanks a lot for your testing and review.
/intergrate

**Pull request closed without merging**
